modules/react-native-status/android/src/main/java/im/status/ethereum/module/Utils.kt

```diff
@@ -153,4 +153,9 @@ class Utils(private val reactContext: ReactApplicationContext) : ReactContextBas
 
         return strArray
     }
+
+    @ReactMethod(isBlockingSynchronousMethod = true)
+    fun validateConnectionString(connectionString: String, callback: Callback) {
+        executeRunnableStatusGoMethod({ Statusgo.validateConnectionString(connectionString) }, callback)
+    }
 }
```

Comment:
Do we need to pass `callback` here if the method is synchronous?

Reply:
Good catch! I missed the Android part when moving to the synchronous method 🙏